src/m365/planner/commands/bucket/bucket-remove.spec.ts

```diff
@@ -407,6 +373,38 @@ describe(commands.BUCKET_REMOVE, () => {
     }));
   });
 
+  it('Correctly deletes bucket by rosterId', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/planner/rosters/${validRosterId}/plans`) {
+        return planResponse;
+      }
+      if (opts.url === `https://graph.microsoft.com/v1.0/planner/plans/${validPlanId}/buckets`) {
+        return singleBucketByNameResponse;
+      }
+
+      throw 'Invalid Request';
+    });
+
+    const deleteStub = sinon.stub(request, 'delete').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/planner/buckets/${validBucketId}`) {
+        return;
+      }
+
+      throw 'Invalid Request';
+    });
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').resolves({ continue: true });
```

Comment:
I think instead of this, we can just pass `confirm: true`.

Reply:
While you can pass `confirm: true`, it doesn't add much since it won't do anything since `result.continue` is undefined
![image](https://github.com/pnp/cli-microsoft365/assets/35696168/5c943f66-98f2-4aea-9a8a-ebf48bfa085f)

Reply:
What I meant is that we can pass `confirm: true` via the command options. It's sufficient to have just 1 test that checks if the confirmation prompt works. No need to have multiple.

Reply:
oh, like that 😅